core/templates/dev/head/domain/skill/SkillUpdateServiceSpec.ts

```diff
@@ -16,28 +16,46 @@
  * @fileoverview Unit tests for SkillUpdateService.
  */
 
+// TODO(YashJipkate) Remove the following block of unnnecessary imports once
+// SkillUpdateService.ts is upgraded to Angular 8.
+import { AudioLanguageObjectFactory } from
+  'domain/utilities/AudioLanguageObjectFactory.ts';
+import { AutogeneratedAudioLanguageObjectFactory } from
+  'domain/utilities/AutogeneratedAudioLanguageObjectFactory.ts';
+import { MisconceptionObjectFactory } from
+  'domain/skill/MisconceptionObjectFactory.ts';
+// ^^^ This block is to be removed.
+
 require('App.ts');
 require('domain/editor/undo_redo/UndoRedoService.ts');
 require('domain/exploration/SubtitledHtmlObjectFactory.ts');
-require('domain/skill/MisconceptionObjectFactory.ts');
 require('domain/skill/SkillObjectFactory.ts');
 require('domain/skill/SkillUpdateService.ts');
 
 describe('Skill update service', function() {
   var SkillUpdateService,
     SubtitledHtmlObjectFactory,
     SkillObjectFactory,
-    MisconceptionObjectFactory,
+    misconceptionObjectFactory,
```

Comment:
Why was this lower cased?

Reply:
The lower case change (similarly elsewhere) is due to the fact that `MisconceptionObjectFactory` is a class and we want this to be a variable to assign the instance to. So, things like `MisconceptionObjectFactory.createFromBackendDict` won't work because MisconceptionObjectFactory is not a constructor as it is a class and TS wants its instance. In other words, TS isn't able to distinguish between the instance `MisconceptionObjectFactory` and the class `MisconceptionObjectFactory` and therefore the differentiation.

Reply:
ah, makes sense!